development/ts-migration-dashboard/scripts/build-app.ts

```diff
@@ -141,16 +141,10 @@ async function copyStaticFiles(src: string, dest: string): Promise<void> {
  * @param options.isInitial - Whether this is the first time this function has
  * been called (if we are watching for file changes, we may call this function
  * multiple times).
- * @param options.isOnly - Whether this will be the only time this function will
```

Comment:
I realize this change probably doesn't belong here, but there are a couple more PRs to go, so I didn't want to delay merging these PRs further. Removing this option means that the `Building dependency tree, hang tight...` message below will appear whenever running `yarn ts-migration:dashboard:build`, which it did not previously.